webauthn-rs-core/src/internals.rs

```diff
@@ -966,6 +966,93 @@ fn tpmtsignature_parser(input: &[u8]) -> nom::IResult<&[u8], TpmtSignature> {
     }
 }
 
+/// From the [TPM Vendor ID Registry][1]
+/// [1]: https://trustedcomputinggroup.org/wp-content/uploads/TCG-TPM-VendorIDRegistry-v1p06-r0p91-pub.pdf,
+pub enum TpmVendor {
```

Comment:
I have a bundle of their CA certs downloaded, which we can add in a seperate PR.